src/views/ModelPlan/ReadOnly/index.tsx

```diff
@@ -260,10 +267,14 @@ const ReadOnly = ({ isHelpArticle }: { isHelpArticle?: boolean }) => {
   const subComponent = subComponents[subinfo];
 
   if ((!loading && error) || (!loading && !data?.modelPlan)) {
-    return <NotFoundPartial />;
+    return <NotFound />;
+  }
+
+  if (!subinfo && !isViewingFilteredGroup) {
+    history.replace(`${location.pathname}/model-basics`);
```

Comment:
Nit: Might be nice to pull `model-basics` into a `const` at the top of this component under something like

```typescript
const defaultSection = "model-basics"
```

just for visibility.